morseCode/morseCodeToWords.test.js

```diff
@@ -0,0 +1,9 @@
+const { mCTWordsResult } = require('./morseCodeToWords');
```

Comment:
We are missing some test scenarios (null input for example)

Also it would be better to throw an error when getting an `undefined` translation instead of returning that result.

Reply:
The test case that returned undefined was removed, that it's not a real scenario, since the only way to get errors is if the input is incorrect, and I build a function to check the input before go through the decode execution.